test/tests/functional/pbs_snapshot_unittest.py

```diff
@@ -94,12 +95,22 @@ def setup_sc(self, sched_id, partition, port):
         :param log_filter: log filter value for the scheduler
         :type log_filter: int
         """
-        a = {'partition': partition,
-             'sched_host': self.server.hostname,
-             'sched_port': port}
+        if sched_priv is not None:
+            a = {'partition': partition,
+                 'sched_host': self.server.hostname,
+                 'sched_port': port,
+                 'sched_priv': os.path.join(sched_priv, 'sched_priv_sc3'),
+                 'sched_log': os.path.join(sched_log, 'sched_logs_sc3')}
+        else:
+            a = {'partition': partition,
+                 'sched_host': self.server.hostname,
+                 'sched_port': port}
         self.server.manager(MGR_CMD_CREATE, SCHED, a, id=sched_id)
         self.scheds[sched_id].create_scheduler()
-        self.scheds[sched_id].start()
+        if sched_priv is not None:
+            self.scheds[sched_id].start(sched_priv)
```

Comment:
Doesn't start() take 'sched_home' as the first argument? why are you providing it sched_priv path?

Reply:
@agrawalravi90 Thanks for the review. Yes, it takes and i am also passing the same just forgot to change the name, will change the name from sched_priv to sched_dir